app/views/pages/hacktoberfest_update.html.erb

```diff
@@ -1,68 +1,78 @@
     
     <h2 class="title_small">What we&rsquo;re doing about it (right now):</h2>
 
-    <p>We’ve traced some of the spammy contributions back to a Hacktoberfest promotional campaign, so we’ve paused all promotion of Hacktoberfest on networks like YouTube, Twitter, and Facebook until we get a handle on this.</p>
+    <p>We’ve traced some of the spammy contributions back to a Hacktoberfest promotional campaign, so <strong>we’ve paused all promotion of Hacktoberfest on networks like YouTube, Twitter, and Facebook</strong> until we get a handle on this.</p>
 
-    <p>Additionally, over the past few years we’ve added values <a href="/details#values">values</a> (<a href="/details#quality">quality over quantity</a>) rules (<a href="/details#spam">no spam allowed</a>) and features (<a href="/details#spam">invalid label</a>) to allow maintainers to flag bad PR&rsquo;s and opt their repos out of the contest entirely, <em>but we need to do more to get at the root of the problem</em></p>
-    
-    <p>We are creating a clear shortcut for these impatient participants to get their shirt and move on, without having to spam repositories on GitHub.</p>
+    <p>Additionally, over the past few years we’ve added <a href="/details#values">values</a> (<a href="/details#quality">quality over quantity</a>), rules (<a href="/details#spam">no spam allowed</a>), and features (<a href="/details#spam">invalid label</a>) to allow maintainers to flag bad PRs and opt their repos out of the contest entirely, but we recognize we need to do more to get to the root of the problem that has emerged:</p>
+
+    <blockquote>Bad participants are making contributions to big open source projects on GitHub that waste maintainers valuable time and resources.</blockquote>
+
+    <p>Since we likely can’t change the behavior of these participants entirely, we are creating a shortcut for these participants to get their shirt and move on, without having to spam repositories on GitHub.</p>
 
-    <p><strong>Here are the specific changes we are working toward to accommodate this:</strong></p>
+    <p>Here are the specific changes we are working toward to accommodate this:</p>
     
     <ol>
-      <li>We are highlighting a "Beginner" path that is clearly marked as the "fastest way to get a T-Shirt" and guides users through creating pull requests on their <em>own repository</em>.</li>
-      <li>We are publishing documentation to make the new path towards completion quick and simple.</li>
-      <li>We are communicating these changes to all participants via website, email, and social media.</li>
+      <li>Recommending a new “shortcut” path towards getting a T-shirt by creating pull requests to your own repository (something we discouraged in the past).</li>
```

Comment:
If you can get a T-Shirt by just making 4 PRs to your own repos, then Hacktoberfest loses all the value it had. Hacktoberfest will no longer be “a program for learning, connecting, and growing”, it will be “who can create a GitHub repository and 4 PRs in it on October 1st”, because this is what will happen: T-shirts are first-come-first-serve, and there’s a limited amount of them, so 70k people will just do the self-PRs and not even bother contributing to other projects, and there won’t be any shirts left over for people who actually want to help open source projects (which you can’t do in one day, and you can make 4 self-PRs in that time).